src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -1,27 +1,68 @@
-import { FullBleed } from "@artsy/palette"
 import { MetaTags } from "Components/MetaTags"
 import { Notifications } from "Components/Notifications/Notifications"
 import { createFragmentContainer, graphql } from "react-relay"
 import { NotificationsApp_me$data } from "__generated__/NotificationsApp_me.graphql"
+import { Media } from "Utils/Responsive"
+import { NotificationsContextProvider } from "Components/Notifications/useNotificationsContext"
+import { NotificationQueryRenderer } from "Components/Notifications/Notification"
+import { useRouter } from "found"
+import { GridColumns, Column, Flex } from "@artsy/palette"
+import { DESKTOP_NAV_BAR_HEIGHT } from "Components/NavBar/constants"
+
+const DESKTOP_HEIGHT = `calc(100vh - ${DESKTOP_NAV_BAR_HEIGHT}px)`
+const MIN_LIST_WIDTH = 340
 
 interface NotificationsAppProps {
   me: NotificationsApp_me$data
 }
 
-const NotificationsApp: React.FC<NotificationsAppProps> = props => {
-  const { me } = props
+const NotificationsApp: React.FC<NotificationsAppProps> = ({ me }) => {
+  const { match } = useRouter()
+
+  const isNotificationsPage = match.location.pathname === "/notifications"
 
   return (
-    <>
+    <NotificationsContextProvider>
       <MetaTags title="Notifications | Artsy" pathname="/notifications" />
 
-      <FullBleed>
-        <Notifications
-          mode="page"
-          unreadCounts={me.unreadNotificationsCount ?? 0}
-        />
-      </FullBleed>
-    </>
+      <Media greaterThan="xs">
+        <GridColumns>
+          <Column
+            span={3}
+            borderRight="1px solid #ddd"
```

Comment:
**super-minor-suggestion**: i would use one of pre-defined colours, in this case I think `black10`

Reply:
oh yes _definitely_. you can specify borderColor as it's own prop

Reply:
I'm going to do dark mode next week — and to do so _all_ hex values need to be banished from this codebase. I'm noticing this sort of thing in a few places. So to be clear: _never_ use a hex value. Always use a system value.